### DISCUSSION TITLE
win11更新（？之后开始菜单自带软件名称显示错误

### DISCUSSION THREAD

**Discussion by mokeling** [zetaloop/ExplorerPatcher#16] [General]
会出现一堆类似ms-resource:AppName/Text的东西，以前也经常出现，不过这次注意了一下是系统更新之后出现的，不确定是否和更新有关，尝试过把相关软件卸载重装但重装之后开始菜单就不显示该软件了

**Comment by zetaloop** [comment]
https://github.com/ZetaSp/ExplorerPatcher/discussions/17
要不咱等等原作者适配更新？
我win10菜单都改不出来了
猜测是微软改的
